src/Symfony/Component/DependencyInjection/Loader/Configurator/DefaultsConfigurator.php

```diff
@@ -66,4 +62,15 @@ final public function instanceof(string $fqcn): InstanceofConfigurator
     {
         return $this->parent->instanceof($fqcn);
     }
+
+    private function validateAttributes(string $tagName, array $attributes, string $prefix = ''): void
+    {
+        foreach ($attributes as $attribute => $value) {
+            if (\is_array($value)) {
+                $this->validateAttributes($tagName, $attributes, $attribute.'.');
```

Comment:
```suggestion
                $this->validateAttributes($tagName, $value, $attribute.'.');
```

I guess we are looking to validate `$value` here, not `$attributes` again? Or I miss something?

(as it's done [here](https://github.com/symfony/symfony/pull/47364/files#diff-942ccd0095dbe8010523f440b62cc19bc9b56357ea3a6f78b5a9e86faf56be38R954))

Reply:
good catch, fixed